src/initialization/FGInitialCondition.cpp

```diff
@@ -1030,18 +1030,21 @@ bool FGInitialCondition::Load(const SGPath& rstfile, bool useStoredPath)
   double version = HUGE_VAL;
   bool result = false;
 
-  if (document->HasAttribute("version"))
+  // If doc has an version, check it. Otherwise fall back to legacy.
+  if (document->HasAttribute("version")) {
     version = document->GetAttributeValueAsNumber("version");
```

Comment:
```suggestion
    double version = document->GetAttributeValueAsNumber("version");
```
Move the declaration of `version` here.